src/frontend/src/pages/TeamsPage/TeamMembersPageBlock.tsx

```diff
@@ -144,7 +193,14 @@ const TeamMembersPageBlock: React.FC<TeamMembersPageBlockProps> = ({ team }) =>
   );
 
   const NonEditingLeadsView = () => (
-    <DetailDisplay label="Leads" content={team.leads.map((lead) => fullNamePipe(lead)).join(', ')} />
+    <Grid container>
+      <Grid item>
```

Comment:
If you make the screen too small, the edit button goes down to the next line (or if you have too many leads assigned), but we want it to stay on the left. you can fix this by setting the xs property on these 2 items